server/bundler/utils.js

```diff
@@ -23,7 +31,36 @@ function pathToRegExp( path ) {
 	return new RegExp( '^' + path + '(/.*)?$' );
 }
 
+function hashFile( path ) {
+	const md5 = crypto.createHash( 'md5' );
+	let data, hash;
+
+	try {
+		data = fs.readFileSync( path );
+		md5.update( data );
+		hash = md5.digest( 'hex' );
+		hash = hash.slice( 0, HASH_LENGTH );
+	} catch ( e ) {
```

Comment:
Super minor but we always use full name for variables.

Reply:
Same remark